cmd/zpool/zpool.d/smart

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+#
+# Show SMART stats
+#
+
+helpstr="
+smart:		Show SMART temperature and important error stats (specific to drive type)
```

Comment:
We need to shorten the 'smart', 'encdev', and 'ses' help messages to fit within the character limit.